pcap-int.h

```diff
@@ -362,7 +375,7 @@ struct oneshot_userdata {
 #define min(a, b) ((a) > (b) ? (b) : (a))
 #endif
 
-int	pcap_offline_read(pcap_t *, int, pcap_handler, u_char *);
+PCAP_API int	pcap_offline_read(pcap_t *, int, pcap_handler, u_char *);
```

Comment:
@guyharris , because the original WinPcap exported this function. To keep the compatibility, I chose to export it in libpcap for Windows too. See this: https://github.com/wireshark/winpcap/blob/master/wpcap/PRJ/WPCAP.DEF#L15